tests/sentry/api/helpers/test_group_index.py

```diff
@@ -156,6 +156,7 @@ def test_unignoring_group(self, send_robust):
         group.refresh_from_db()
 
         assert group.status == GroupStatus.UNRESOLVED
+        assert group.substatus is GroupSubStatus.ONGOING
```

Comment:
Should unresolving an ignored/archived issue set the `substatus` to `ONGOING` or none?

Reply:
It should be ONGOING